python/ql/lib/semmle/python/RegexTreeView.qll

```diff
@@ -751,6 +767,9 @@ class RegExpGroup extends RegExpTerm, TRegExpGroup {
    */
   int getNumber() { result = re.getGroupNumber(start, end) }
 
+  /** Holds if this is a capture group. */
+  predicate isCapture() { not exists(this.getNumber()) }
```

Comment:
I may be completely misunderstanding this predicate, but is that `not` correct? Is this not expressing "not an unnamed capture group" instead?

Reply:
I believe all capture groups have a number, but only some have a name. Is that assumption incorrect?

Reply:
Oh wait, the `not` makes no sense at all. You're right.